bittensor/wallet.py

```diff
@@ -140,6 +142,10 @@ def add_args(cls, parser: argparse.ArgumentParser, prefix: str = None):
         """
         prefix_str = "" if prefix == None else prefix + "."
         try:
+            # TODO: Where should the default values be stored? Currently, they are scattered all across the codebase.
+            #  Maybe i would create a constant in this class and return it as config. Later we could use something
+            #  like self.config.wallet.name to set the default value on the parser.
+            #  ^ Look at the comment in the config method.
             default_name = os.getenv("BT_WALLET_NAME") or "default"
```

Comment:
I wasn't aware that they were already using a BT_ convention for their envs. If that's consistent elsewhere in the codebase, let's update our config env loader to use the same BT_ convention instead of BTCLI_.